src/main/resources/config/liquibase/changelog/20230705141500_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20230705141500" author="kotzian">
+        <addColumn tableName="iris_session">
+            <column name="creation_date" type="datetime">
+                <constraints nullable="false" />
```

Comment:
This will cause problems with already existing sessions.
You'll have to assign a default value to all existing sessions (but not a default for all future sessions)